### ISSUE TITLE
Ability to cancel my Coffee order 

### ISSUE THREAD

**Issue by nemccarthy** [assafgelber/slash-coffee#6]
As a user it would be great to cancel a coffee order - having this feature would also solve issue #1 as you could just cancel and replace the order if you wanted to change it.

**Comment by assafgelber** [comment]
I agree that this would be a great feature, unfortunately I don't have the capacity to build it myself any time soon, but if you want to take a stab at it, I'd be glad to help out!
